### PR TITLE
feat: remove enrollments entitlements and license manager from old view

### PR DESCRIPTION
**Pull request by Ali-D-Akbar**
[PROD-2757](https://openedx.atlassian.net/browse/PROD-2757)
We've started the deprecation for the old Support Tools MFE in a number of phases. This is the first PR of the deprecation process and it removes old components like enrollment, entitlement and license manager. 

Screenshot 2022-03-30 at 12 59 14 AM

### CODE REVIEW

**Comment by azanbinzahid** [comment]
Shouldn't we remove v2 toggle button along with it?

File: src/users/UserPage.jsx [hunk 1]
```diff
@@ -37,9 +34,6 @@ export default function UserPage({ location }) {
   const [searching, setSearching] = useState(false);
```
Comment (DawoudSheraz) [inline src/users/UserPage.jsx:34]: don't we need to delete this file too?

**Comment by DawoudSheraz** [comment]
+1 to what @azanbinzahid has said. The toggle should be removed too. Then, in the follow-up PR, the URLs would need an update where all the references to `v2` would be removed. But do not merge this PR until the relevant parties have been notified.

**Review by DawoudSheraz — APPROVED** [review]
didn't test it out on the local. Straightforward removal.

**Review by azanbinzahid — APPROVED** [review]

**Pull request merged**